.circleci/create_circleci_config.py

```diff
@@ -226,16 +226,16 @@ def to_dict(self):
             test_command += " || true"
         steps.append({"run": {"name": "Run tests", "command": test_command}})
 
-        check_test_command = f'if [ -s reports/tests_{self.name}/failures_short.txt ]; '
+        check_test_command = f'if [ -s reports/{self.job_name}/failures_short.txt ]; '
```

Comment:
The `test_` is added to `self.name` in `self.job_name` based on a rule. `self.jon_name` is the one that is used when these files are created.